test/service/test_manifest.py

```diff
@@ -440,14 +440,43 @@ def test_manifest_zarr(self):
                 self.assertEqual(200, response.status_code)
                 lines = response.content.decode().splitlines()
                 file_prefix = 'output="587d74b4-1075-4bbf-b96a-4d1ede0481b2/'
+                location_prefix = f'url="{config.service_endpoint()}/repository/files'
+                file_location_postfix = '"'
```

Comment:
> Was thrown off by the unpaired doublequote, might benefit from a comment but YMMV
@noah-aviel-dove I added `file_location_postfix` to clear it up a bit, but not sure if it is effective? LMK

Reply:
I don't think this is an improvement. I'll approve the PR but I think you should revert this to what you had originally before assigning to Hannes. It was probably overzealous of me to draw attention to this.